portfolio/src/main/webapp/script.js

```diff
@@ -18,12 +18,11 @@ function displayComment(limValue){
         fetchString += "?limit=";
         fetchString += limValue;
     }
-    commentContainer = document.getElementById("comment_container");
-    commentContainer.innerHTML = "";
     fetch(fetchString).then(response => response.json()).then(data => {
         for(var i = 0; i < data.length; ++i){
+            commentContainer = document.getElementById("comment_container");
+            commentContainer.innerHTML = "";
```

Comment:
ah sorry i meant to say to move this to be after the fetch but before the for loop. Otherwise, we'll only ever display the last element because we keep clearing in innerHTML of the comment_container before writing something new to it. We can also move the obtaining of the commentContainer to after fetch but before the for loop as well so we only make the call to get it once.

Reply:
ohhh yea I'm not sure why I was not thinking when I changed this. I'll fix it